packages/config/src/layer2s/aztec.ts

```diff
@@ -162,12 +243,13 @@ export const aztec: Layer2 = {
         'AztecFeeDistributor',
         'Contract responsible for distributing fees and reimbursing gas to Rollup Providers.',
       ),
-      {
-        address: EthereumAddress('0xd3a6D9De4cbC2CC7529361941e85b1c3269CcBb1'),
-        description:
-          'Turbo Plonk zkSNARK Verifier. It can be upgraded by the owner with no delay.',
-        name: 'TurboVerifier',
-      },
+      discovery.getContractDetails('TurboVerifier', {
+        description: 'Turbo Plonk zkSNARK Verifier.',
+        upgradableBy: ['Aztec Multisig'],
+        upgradeDelay: 'No delay',
+        upgradeConsiderations:
+          'Verifier field in RollupProcessor can be changed with no delay.',
+      }),
       {
         address: EthereumAddress('0x7FaE73Be814d94318fa0756a5D73ae9cf3BA7530'),
```

Comment:
I'm not sure if it's necessary to show this address, this is just a library... WYT @lucadonnoh

Reply:
i think it's important to show where the verification keys are